examples/using-guidebot-flags.md

```diff
@@ -97,7 +101,6 @@ exports.help = {
   name: 'role',
   category: 'Moderation',
   description: 'It can give or take a role to/from a user.',
-  usage: 'ping'
+  usage: 'role <-add|-remove> <role name>'
 };
   ```
```

Comment:
This indentation feels off. It may break the markdown parser.

Reply:
The indentation should be fine, I've viewed all of my changes in both VSCode's previewer, and on the repo itself. Is there an indentation that you recommend? I wrote this in accordance with how Guidebot will want the command to be indented.